llama.cpp

```diff
@@ -58,6 +59,7 @@ static const size_t MB = 1024*1024;
 static const std::map<e_model, size_t> & MEM_REQ_SCRATCH0()
 {
     static std::map<e_model, size_t> k_sizes = {
+        { MODEL_3B,    128ull * MB },
```

Comment:
I was suspicious here, since its so much less. But it ran without any issue for me, so i guess the others might be too large. :)

Reply:
I tested before merging with `--no-mmap` f16 model `--memory-f32` `-c 2048` `-n 4096` which should be the worst case but it worked